recipes-wpe/wpeframework/include/wpeframework-common.inc

```diff
@@ -1,24 +0,0 @@
-# Common for thunder and modules which depends thunder
```

Comment:
What is the reason to remove the file?

Reply:
Now the thunderTools is moved to new repo, hence the same SRC details we cannot use for both thunder and tools now. Now all the required SRC settings are moved to corresponding recipe instead keeping in single place

Reply:
added